src/app/tests/TestReadInteraction.cpp

```diff
@@ -576,7 +576,6 @@ void TestReadInteraction::TestReadClientGenerateOneEventPaths(nlTestSuite * apSu
     NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
 
     chip::app::EventPathParams eventPathParams[2];
```

Comment:
Here is the mock should be used -- please add round trip tests for reading wildcard events.

Reply:
updated to test with one wildcard eventId which would be interpreted into two paths in handler side